### DISCUSSION TITLE
Mobile app

### DISCUSSION THREAD

**Discussion by BrentMifsud** [benawad/dogehouse#1225] [General]
Need an iOS dev? I can help throw something together if you guys have the designs for it.

**Comment by BrentMifsud** [comment]
nvm I see you guys already have a flutter app going. My flutter knowledge is super basic.

**Reply by ghost** [reply]
so It's better to start with dart and when you got comfortable with It I believe you could understand a part of that but nut much yet so in such case youtube videos could be such great idea
